### PR TITLE
mandatory category test finished

### PR DESCRIPTION
**Pull request by Andresgarciasch**

### CODE REVIEW

**Comment by Andresgarciasch** [comment]
https://github.com/4GeeksAcademy/About-4Geeks-Academy/issues/3622

File: breathecode/registry/serializers.py [hunk 1]
```diff
@@ -336,10 +336,19 @@ def create(self, validated_data):
...
-            **validated_data, 'academy': academy,
-            'readme_raw': readme_raw
-        })
+        try:
+            return super(PostAssetSerializer, self).create({
+                **validated_data, 'academy': academy,
+                'readme_raw': readme_raw
+            })
+        except Exception as e:
+            print(dir(e))
```
Comment (jefer94) [inline breathecode/registry/serializers.py:345]: remove prints

**Comment by jefer94** [comment]
Pull the changes from development branch and fix the tests that appear with `pipenv run ptest`

File: breathecode/registry/tests/urls/tests_academy_asset.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            'category': ['This field cannot be blank.'],
+            'status_code': 400,
+        }
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(self.bc.database.list_of('registry.Asset'), [])
+
+
+#class RegistryTestAsset2(TestCase)
```
Comment (jefer94) [inline breathecode/registry/tests/urls/tests_academy_asset.py:65]: Remove the comment

File: breathecode/registry/tests/urls/tests_academy_asset.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        response = self.client.post(url, data, format='json')
+        json = response.json()
+        expected = {
+            'category': ['This field cannot be blank.'],
+            'status_code': 400,
+        }
+
+        self.assertEqual(json, expected)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertEqual(self.bc.database.list_of('registry.Asset'), [])
```
Comment (jefer94) [inline breathecode/registry/tests/urls/tests_academy_asset.py:62]: Write tests until a new asset was created

**Review by jefer94 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**